src/Symfony/Component/AutoMapper/Attributes/MapIf.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Symfony\Component\AutoMapper\Attributes;
+
+use Attribute;
+
+/**
+ * Condition whether to map a property.
+ *
+ * @experimental
```

Comment:
AFAIK we cannot have an experimental component in 6.4 as it is a LTS, it should target 7.0 in any case.

Reply:
done thanks

Reply:
Also experimental should not be the default. Previous art proves that automatic object mapping is not controversial so I don't think it is justified here, and this would need adoption. Let's first to bring components with good enough public APIs first

Reply:
I thought that everything introduced should be experimental as it gives us time to adapt.